pkgs/by-name/ll/llama-cpp/package.nix

```diff
@@ -29,21 +32,24 @@ let
 in
 effectiveStdenv.mkDerivation (finalAttrs: {
   pname = "llama-cpp";
-  version = "1848";
+  version = "1892";
 
   src = fetchFromGitHub {
     owner = "ggerganov";
     repo = "llama.cpp";
     rev = "refs/tags/b${finalAttrs.version}";
-    hash = "sha256-KuomiKU9c06Ux/ZcqctFdPQykGtjDzArN+tElPJVQ60=";
+    hash = "sha256-FNyl8bR0rg6cixcqidqzk9rG62+CI/0BNvzHuUkBq1E=";
   };
 
   postPatch = ''
     substituteInPlace ./ggml-metal.m \
       --replace '[bundle pathForResource:@"ggml-metal" ofType:@"metal"];' "@\"$out/bin/ggml-metal.metal\";"
   '';
 
-  nativeBuildInputs = [ cmake ] ++ lib.optionals blasSupport [ pkg-config ] ++ lib.optionals cudaSupport [
+  nativeBuildInputs = [
+    cmake
+    git # not used for anything just included to silence a warning
```

Comment:
Adding this actually makes more noise in the logs, it removes the "git not found" warning and adds two "fatal: not a git repository".

Reply:
the fatal not a git repository log is already there if I'm not wrong.

Reply:
ah just checked and there are 2 other git warnings. I tend to agree with you, this doesn't really improve anything.